### ISSUE TITLE
Feature request: Auto launch mass:/boot.elf if present

### ISSUE THREAD

**Issue by SpaceAgeHero** [CTurt/FreeDVDBoot#172]
Would it be possible to add this feature?
Auto launch mass:/boot.elf if present, otherwise fall back to uLaunchELF on disc.
I think this would open up a door to more flexibility.
Me for instance I like to keep my OPL up to date on USB storage and would like to launch it as fast as possible.

**Comment by grbolivar** [comment]
+1 for this.

**Comment by ghost** [comment]
Good idea . Now you can config uLE to boot it with a specific button . It will save a ULAUNCHELF.CNF on mass but the problem is that it won't autoload it when rebooting .

**Comment by Pillepip** [comment]
any solution to this? I'm trying to auto launch OPL as well, only using the disc + USB
